### PR TITLE
[2.7] Removed 2.5 versionadded as its deprecated

### PR DESCRIPTION
**Pull request by wouterj**

### CODE REVIEW

**Comment by xabbuh** [comment]
If I don't misunderstand our workflow, this will only happen when 2.7 is actually released.

File: cookbook/console/console_command.rst [hunk 1]
```diff
@@ -194,44 +194,4 @@ instead of
...
-                    'name'    => 'Fabien',
-                    '--yell'  => true,
-                )
-            );
-
-            $this->assertRegExp('/.../', $commandTester->getDisplay());
-
-            // ...
-        }
-    }
```
Comment (xabbuh) [inline cookbook/console/console_command.rst]: Seems like you removed too much.

**Comment by WouterJ** [comment]
@xabbuh see https://github.com/symfony/symfony-docs/issues/4774 :

 > Remove all versionadded directives - and any other notes related to features changing or being new - for the version (e.g. 2.1) from the master branch. The result is that the next release (which is the first that comes entirely after the end of maintenance of this branch), will have no mentions of the old version (e.g. 2.1).

As 2.5 reaches end of maintance this month and 2.7 is released in May, it should not contain any versionadded directives for 2.5. Stricly spoken, we should merge this PR in january, but I think we can think ahead and merge it already.

File: cookbook/console/console_command.rst (hunk 1 above)
Comment (WouterJ) [inline cookbook/console/console_command.rst]: yeah, my script has problems with code examples at the end of a file :)

**Comment by weaverryan** [comment]
:thumbsup: then we can continue to clarify all of this on your issue - it's always been a little confusing (pretty sure I wrote these confusing-ish rules)

**Pull request merged**